src/Codeception/Command/Run.php

```diff
@@ -3,8 +3,6 @@
 
 use Codeception\Codecept;
 use Codeception\Configuration;
-use Codeception\Lib\GroupManager;
-use Codeception\Util\PathResolver;
 use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputArgument;
 use Symfony\Component\Console\Input\InputInterface;
```

Comment:
Please document new syntax in `Usage` section of class level docblock in this file.  (Line 24)